book/src/SUMMARY.md

```diff
@@ -39,6 +39,7 @@
   * [Installation](paper-wallet/installation.md)
   * [Creating and Using a Seed Phrase](paper-wallet/keypair.md)
   * [Paper Wallet Usage](paper-wallet/usage.md)
+* [Offline Signing](offline-signing/README.md)
```

Comment:
I think this needs to be indented to nest beneath Paper Wallet, assuming that's your intention..?

Reply:
Paper Wallet and Offline Signing are really tangential features.  They work well together, but can also be used independently.  I intend to have the durable nonce docs live under Offline Signing, too.

Reply:
Ah, didn't realize this was the start of a new section. Seems sort of odd as a top-level section, but I guess we don't have a better org structure right now. I'll noodle on that.